docs/nilup.md

```diff
@@ -1,6 +1,30 @@
 # nilup
 
-`nilup` is the tool to manage Nillion SDK versions.
+`nilup` is an installer and version manager for the Nillion SDK
+
+To install `nilup`, run: `curl https://nilup.nilogy.xyz | bash`
+
+If you install the Nillion SDK using `nilup`, all the Nillion SDK commands will be managed by `nilup`
+and it will pick the version to use base on the next order.
```

Comment:
```suggestion
If you install the Nillion SDK using `nilup`, all the Nillion SDK commands will be managed by `nilup`,
which will pick the version to use based on the next order.
```

also, not sure what "based on the next order" means here? could you try clarifying? :)